ee/agent/keys.go

```diff
@@ -39,18 +40,21 @@ func SetupKeys(logger log.Logger, store types.GetterSetterDeleter) error {
 		return fmt.Errorf("setting up local db keys: %w", err)
 	}
 
-	err = backoff.WaitFor(func() error {
-		hwKeys, err := setupHardwareKeys(logger, store)
+	// Secure Enclave is not currently supported, so don't spend startup time waiting for it to work -- see keys_darwin.go for more details.
+	if runtime.GOOS != "darwin" {
```

Comment:
might be simpler as `if runtime.GOOS == "darwin" { return nil }`

Reply:
Ah yup that reads much better, thanks -- updated